<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidgetFileList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>File list</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAddFiles">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Add files</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelFiles">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete files</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCalculate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>201</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMagR">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>247</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Magnificant R</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMagR">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelMagG">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>247</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Magnificant G</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMagG">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelMagB">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>247</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Magnificant B</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMagB">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelMagR_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Usage  data in Pix file</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxUsePhaseNumber">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelMagR_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Phase</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxUsePageNumber">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelMagR_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Page</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
